rpc/grpc/config.go

```diff
@@ -29,9 +29,9 @@ type Config struct {
 	// Endpoint is an address of GRPC netListener
 	Endpoint string `json:"endpoint"`
 
-	// SocketType defaults to "tcp" if unset, and can be set to one of the following values:
-	// "tcp", "tcp4", "tcp6", "unix", "unixpacket"
-	SocketType string `json:"socket-type"`
+	// Network defaults to "tcp" if unset, and can be set to one of the following values:
+	// "tcp", "tcp4", "tcp6", "unix", "unixpacket" or any other value accepted by net.Listen
+	Network string `json:"socket-type"`
```

Comment:
Missed the struct tag.

Reply:
Good catch, should be fixed in next push :)